### DISCUSSION TITLE
（偶然事件）部分弹幕显示错误

### DISCUSSION THREAD

**Discussion by SANA1096** [ywmoyue/biliuwp-lite#451] [bug]
### Bug 描述

最新arm版应用，在一个视频中发现部分弹幕持续显示错乱问题，未于其他视频中复现bug，网页及手机端显示正常
弹幕库为NSDanmuku，视频链接为https://b23.tv/BV1gU4y187xA，部分弹幕设置附于截图

### 操作步骤

```plain text
正常视频播放步骤
```

### 期望结果

问题得到改善

### 截图

![屏幕截图 2023-12-23 144058](https://github.com/ywmoyue/biliuwp-lite/assets/153289945/a6d3055c-05b0-4230-8a11-0bf753e50ae4)
![屏幕截图 2023-12-23 144243](https://github.com/ywmoyue/biliuwp-lite/assets/153289945/bf0a4fff-0057-4911-8ca9-7df4d3a505ce)
![屏幕截图 2023-12-23 145111](https://github.com/ywmoyue/biliuwp-lite/assets/153289945/fc0a1c9f-3c11-41eb-a011-22b323c00ed1)

### 日志

[20231223.log](https://github.com/ywmoyue/biliuwp-lite/files/13758278/20231223.log)

### 应用来源

Github/其他Github镜像站

### 安装方式

脚本安装

### 应用架构

Arm64

### 代码贡献

我更擅长发现问题

**Comment by GD-Slime** [comment]
可能是高级弹幕? 使用寒霜弹幕库可能可以解决此问题.

**Reply by SANA1096** [reply]
确实换了寒霜弹幕库没有显示错误了，但是寒霜弹幕库好像弹幕内容跟NSDanmuku显示的不太一样？数量也不太一致，还有弹幕数量多时过分重叠问题

**Reply by GD-Slime** [reply]
用高级弹幕艺术的视频测试了，目前似乎NSDanmuku在解析高级弹幕时会出现问题。有高级弹幕需求时先使用寒霜弹幕库吧。
